package.json

```
@@ -0,0 +1,82 @@
+{
+	"name": "tldraw-server",
+	"type": "module",
+	"version": "0.0.1",
+	"license": "AGPL-3.0",
+	"description": "tldraw backend",
+	"author": "Schulcloud-Verbund",
+	"private": true,
+	"engines": {
+		"npm": ">=8.0.0",
+		"node": ">=20.0.0"
+	},
+	"scripts": {
+		"build": "nest build",
+		"format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+		"start": "nest start server",
+		"start:server:dev": "nest start server --watch",
+		"start:server:debug": "nest start server --debug --watch",
+		"start:server:prod": "node  --env-file .env dist/apps/tldraw-server.js",
```

```suggestion
		"start:server:prod": "node  --env-file=.env dist/apps/tldraw-server.js",
```
Funktioniert auch ohne Gleichheitszeichen, aber ich finds deutlicher und in der Doku stehts auch so: https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs